### DISCUSSION TITLE
[ISSUE] Unholy Death Knight Death and Decay AOE Combo

### DISCUSSION THREAD

**Discussion by mikemangu** [Hekili/hekili#812] [General]
@Hekili 
So I've been maining Unholy Death Knight and the AOE recommendations seem to be off.

For example, if 3+ mobs come out, the highest burst dps is, Drop Death and Decay (if not up), Outbreak (Spread the rona), and Epedemic until everything blows up, also some combination of Unholy Blight.

However, it seems to take on a Single DPS rotation with Death Coils instead of the higher DPS burst option. 

I understand that this is coming from SimC profile, but the action lists is showing multiple AOE options and based on different scenarios, however I find myself having to drop Death and Decay regardless of the Recommendation.

I saw the latest update made some changes to "Adjustments to DnD/Defile/Deaths Due to only count targets who are expected to live long enough to matter (aoe_setup #1)." but this seems to either have broken the rotation for AOE or made it worst.

**Comment by Hekili** [comment]
Please follow ticket instructions.
